db/migrate/20160719005841_create_categorisations.rb

```diff
@@ -0,0 +1,27 @@
+class CreateCategorisations < ActiveRecord::Migration
+  def up
+    create_table :categorisations do |t|
+      t.references :category, foreign_key: true, null: false
+      t.references :post, foreign_key: true, null: false
+      t.integer :order, null: false
+
+      t.index %i(post_id category_id), unique: true
+
+      t.timestamps null: false
+    end
+
+    Post.all.each do |post|
```

Comment:
Shouldn't reference model from migration.
http://qiita.com/nay3/items/ef773006cd7f815a07cd#2-%E3%82%A2%E3%83%97%E3%83%AA%E3%82%B1%E3%83%BC%E3%82%B7%E3%83%A7%E3%83%B3%E3%81%A7%E5%AE%9A%E7%BE%A9%E3%81%97%E3%81%9F%E3%82%AF%E3%83%A9%E3%82%B9%E3%82%92%E7%9B%B4%E6%8E%A5%E4%BD%BF%E3%82%8F%E3%81%AA%E3%81%84